### PR TITLE
complete the changelog file for 1.7

### PR DESCRIPTION
**Pull request by xabbuh**
I forgot to mark #1449 as "WIP". So this PR adds the missing pieces.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
 -----
 
+* remove uses of the reflection API in favor of faster solutions when possible
+
+* fixed the configuration to use serialization groups and versions at the same time
+
 1.7.7
 -----
 
+* do not call methods on Symfony 3.x installations that have been removed in Symfony 3.0
```
Comment (javiereguiluz) [inline CHANGELOG.md]: I don't understand this phrase. Does it mean: *"do not call methods that have been removed in Symfony 3"* ?

Comment (xabbuh) [inline CHANGELOG.md]: Yeah, I reworded this a bit. Do you think it's more clear now?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,25 @@
 CHANGELOG
 =========
 
+* fixed the `AllowedMethodsRouteLoader` to work with Symfony 3.0
+
 1.7.8
 -----
 
+* remove uses of the reflection API in favor of faster solutions when possible
```
Comment (Ener-Getick) [inline CHANGELOG.md]: we should always use the past right ?

Comment (xabbuh) [inline CHANGELOG.md]: good catch, fixed

**Comment by Ener-Getick** [comment]
LGTM :+1:

**Pull request merged**